packages/haul-preset-0.59/src/defaultConfig.ts

```diff
@@ -15,7 +15,8 @@ import getBabelConfigPath from './getBabelConfigPath';
 
 export default function getDefaultConfig(
   runtime: Runtime,
-  options: EnvOptions
+  options: EnvOptions,
+  inlineSourceMap?: boolean
```

Comment:
How about we just pass a whole `haulConfig` here?

Reply:
Good idea :+1: